core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1433,8 +1433,13 @@ public static function sendFileToBrowser($strFile, $inline=false)
 	 */
 	public static function loadDataContainer($strTable, $blnNoCache=false)
 	{
+		if (\func_num_args() > 1)
+		{
+			trigger_deprecation('contao/core-bundle', '4.13', 'Calling "%s" with the $blnNoCache parameter has been deprecated and will no longer work in Contao 5.0.', __METHOD__);
+		}
+
 		$loader = new DcaLoader($strTable);
-		$loader->load($blnNoCache);
+		$loader->load(...($blnNoCache ? array(true) : array()));
```

Comment:
Is this really necessary? We can continue to pass `$blnNoCache` I think.

Reply:
Passing the parameter causes a deprecation now because of `func_num_args()`

Reply:
Of course. 🤦‍♂️